app/models/accounting/qb/transaction_reconciler.rb

```diff
@@ -30,6 +33,10 @@ def reconcile(transaction)
 
       attr_reader :qb_connection, :principal_account, :qb_division
 
+      def handle_qb_intuit_request_exception(txn, e)
+        raise IntuitRequestError.new(transaction: txn, message: e.message)
```

Comment:
At first glance it's not clear what the difference is between IntuitRequestError and IntuitRequestException. It seems like the former wraps the latter and adds a transaction object for the error handler to use? Maybe call it WrappedIntuitRequestException or AnnotatedIntuitRequestException?